pages/index.vue

```diff
@@ -265,7 +231,21 @@ export default {
   },
   data() {
     return {
-      loading: true
+      loading: true,
+      coordinators: [
+        [
+          'Aman Gupta',
+          'https://lh6.googleusercontent.com/-Nabz6JJKsqc/AAAAAAAAAAI/AAAAAAAAAoI/biOxaDpjiec/photo.jpg'
+        ],
+        [
+          'Harshul Gupta',
+          'https://res.cloudinary.com/amuroboclub/image/upload/v1529768915/profile_img/IMG_20180328_004220805.jpg'
+        ],
+        [
+          'Kakul Shrivastava',
+          'https://res.cloudinary.com/amuroboclub/image/upload/v1543928884/profile_img/FB_IMG_1543928797162.jpg'
+        ]
```

Comment:
Make it a list of object

```
{
  name: '',
  avatar: ''
}
```

What's the logic of making it a matrix?

Reply:
> What's the logic of making it a matrix?

Looks good

Should I make it a list of object?

Reply:
Yes

Reply:
Looks good, reads terrible

Reply:
Okay I will make it a list of object